core-scene2d/src/de/golfgl/gdx/controllers/ControllerMenuStage.java

```diff
@@ -103,7 +114,7 @@ public void clearFocusableActors() {
      */
     public void removeFocusableActorsNotOnStage() {
         for (int i = focusableActors.size - 1; i >= 0; i--) {
-            if (focusableActors.get(i).getStage() != this)
+            if (focusableActors.get(i) == null || focusableActors.get(i).getStage() != this)
```

Comment:
I wonder how there can be null-entries in the array? Maybe this is obsolete when we have the null check on addFocusableActor?

Reply:
You can still modify actors while they are in the array and make em null. Which means later when you call that method you get a nullpointer exception, this will still remove it without causing the exception. It would be bad design but it is a fail safe. I could not find other places where it could potential crash with a nullpointer exception. You could leave it out but I learned that some people make some crazy at times. Better safe than sorry imho